srcbmi/mf6bmi.f90

```diff
@@ -830,13 +809,7 @@ function get_model_name(grid_id) result(model_name)
       end if
     end do
     
-    write(error_msg,'(a,i6)') 'BMI error: no model for grid id ', grid_id
-    call sim_message(error_msg, iunit=istdout, skipbefore=1, skipafter=1)
-    
-    
+    write(error_msg,'(a,i6)') 'BMI error: no model for grid id ', grid_id
```

Comment:
Unless these is a reason to do otherwise we are using `i0` and `g0` for integers and floats.

Reply:
Done.